benefits/locale/es/LC_MESSAGES/django.po

```diff
@@ -28,6 +28,13 @@ msgstr ""
 "may need to renew. Using this benefit means your new transit fare is half of "
 "the standard fare."
 
+#, fuzzy
+#| msgid "eligibility.pages.start.mst_login.headline"
```

Comment:
What are these fuzzy IDs for?

Reply:
Remedying this duplication in next commit

Reply:
When you run the `bin/makemessages.sh` -- make sure to look for these before committing. If `gettext` doesn't quite know what to do with a `msgid` it adds a fuzzy entry. Usually it's because something was renamed or removed; often it's a simple fix of uncommenting the `msgid` and deleting the fuzzy comment.